test_pull_requests

```diff
@@ -1155,9 +1328,20 @@ popd
 
           branch = req['base']['ref']
 
+          # we only want to consider pull requests into branches we care about
           if $branches.include?(branch) || $branches.include?('*')
 
             $stderr.puts "  Updated at: #{req['updated_at']}"
+            # we only want to consider pull requests that have been modified in
+            # the last twelve hours, to stop us from doing extra work when we
+            # don't need to. Also, just to ensure that we don't forget a pull
+            # request forever on accident, there is a 10% chance we'll consider
+            # a pull request even if it is inactive
+            #
+            # TODO: wouldn't this be better if we brought this time down to
```

Comment:
Remove.  Way too specific for this repo which is more generic than the case you are talking about.